app/subscriber/src/features/filter-media/constants/MediaFilterTypes.ts

```diff
@@ -0,0 +1,9 @@
+export enum MediaFilterTypes {
```

Comment:
Is there a reason to have an enum for these?  We have a look up which provides all of our media types and it will never have an issue if the names change.

Reply:
I second this.